.github/ISSUE_TEMPLATE/implementation-ticket.yml

```
@@ -62,6 +62,6 @@ body:
     attributes:
       label: Context
       description: |
-        Provide the "why", motivation, and alternative approaches considered -- linking to previous refinement issues, spikes, Notion docs as appropriate
-    validations:
-      required: false
+        Provide the "why", motivation, and alternative approaches considered -- linking to previous refinement issues, spikes and documentation as appropriate
+      validations:
+        required: false
```

This is still indented too much.  After you merge in your changes, you should view the file and it should render with text boxes.  Right now it shows an error at the top that this key isn't allowed.
```suggestion
    validations:
      required: false
```